hudi-common/src/main/java/org/apache/hudi/common/fs/inline/InLineFsDataInputStream.java

```diff
@@ -33,11 +33,11 @@
  */
 public class InLineFsDataInputStream extends FSDataInputStream {
 
-  private final int startOffset;
+  private final long startOffset;
   private final FSDataInputStream outerStream;
   private final int length;
```

Comment:
length need be `long` too

Reply:
**length** be **int** type is ok,  which in the FSDataInputStream(Hadoop) is also **int**.   **length** as maximum number of bytes to read be **int** type may by enough.

Reply:
in `InLineFSUtils.getInlineFilePath`, the inLineLength is `long`, so I think it's better to be consistent.

Reply:
> in `InLineFSUtils.getInlineFilePath`, the inLineLength is `long`, so I think it's better to be consistent.

it make sense. thx.